tests/test_sensor.py

```diff
@@ -344,3 +345,23 @@ async def test_camera_unique_id(
     registry_entry = er.async_get(hass).async_get(entity_id)
     assert registry_entry
     assert registry_entry.unique_id == unique_id
+
+
+async def test_binary_sensor_all_can_be_enabled(hass: HomeAssistant) -> None:
```

Comment:
Typo: Should be 'sensor' not 'binary_sensor'

Reply:
Darn copy pasta got me